src/views/accounts/account-details/account-details.tsx

```diff
@@ -34,13 +27,6 @@ export function AccountDetails() {
   const [activeTabIdx, setActiveTabIdx] = React.useState(0)
   let tabs = Object.values(TabNames).slice(0, hasMultisig ? undefined : -1)
   const activeTabName = tabs[activeTabIdx]
-  const [, setContainerProps] = usePageContainerProvider()
-
-  React.useLayoutEffect(() => {
-    setContainerProps({
-      w: { base: "full" },
-    })
-  }, [setContainerProps])
```

Comment:
This didn't seem to be doing anything. We don't even use PageContainer on Home so it may not have been fully implemented or something.